src/main/resources/assets/miningmaster/textures/item/mm_tab_icon.png.mcmeta

```diff
@@ -0,0 +1,18 @@
+{
+  animation {
+    interpolate true,
+    frametime 48,
+    frames [
+      0,
+      1,
+	  2,
+	  3,
+	  4,
+	  5,
+	  6,
+	  7,
+	  8,
+	  9
+    ]
+  }
+}
```

Comment:
This isn't even a valid .mcmeta file??

Reply:
bro what
I pulled the formatting straight from soul salt in IE 😭

Reply:
> bro what I pulled the formatting straight from soul salt in IE sob

and the soul salt isn't animated iirc

> The animation is then controlled using a `.mcmeta` file in JSON format with the same name

https://minecraft.fandom.com/wiki/Resource_Pack#Animation

Reply:
it literally is?? it cycles between two shades of blue from an MCmeta file